docs/config/README.md

```diff
@@ -2284,6 +2285,41 @@ disabled = false
 show_always = true
 ```
 
+## Vagrant
+
+The `vagrant` module shows the currently installed version of Vagrant.
+The module will be shown if any of the following conditions are met:
+
+- The current directory contains a `Vagrantfile` file
+
+### Options
+
+| Option     | Default                            | Description                                         |
+| ---------- | ---------------------------------- | --------------------------------------------------- |
+| `format`   | `"via [$symbol$version]($style) "` | The format for the module.                          |
```

Comment:
We have been changing the default format string to handle failures if fetching the version fails for some other reason more gracefully.
```suggestion
| `format`   | `"via [$symbol($version )]($style)"` | The format for the module.                          |
```